packages/image_picker/android/build.gradle

```diff
@@ -21,7 +21,7 @@ buildscript {
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.3.0'
+        classpath 'com.android.tools.build:gradle:3.4.1'
```

Comment:
Seems not related to this PR?

Reply:
gradle was updated after the PR made, it would be better if added with this PR else not necessary.

Reply:
ok, it'd be nice to rebase the PR so we won't see this diff here

Reply:
Sorry i misunderstood your comment here. I think what I wanted to say is that let's remove change in this particular PR, since it is not related to the PR. Also please remove the related bit in the CHANGELOG

Reply:
ok, I'll remove gradle change and comment from the changelog.

Reply:
@cyanglaz I've restored the gradle version and updated changelog.